test/powershell/Modules/Microsoft.PowerShell.Utility/string.tests.ps1

```diff
@@ -9,6 +9,11 @@
 
             $fileNameWithDots = $fileName.FullName.Replace("\", "\.\")
 
+            $subDirName = Join-Path $TestDrive 'selectSubDir'
+            New-Item -Path $subDirName -ItemType Directory > $null
```

Comment:
Suggest using -ErrorAction Stop.

Reply:
I see we usualy use `-Force` - I added.
Please clarify about `-ErrorAction Stop` - I don't found such samples in our tests.

Reply:
For me, its all about failure diagnosability.  The test is dependent on the directory and if new-item fails, there's no point in the test continuing.  Using -EA Stop ensures you get the error at the point of the failure instead of later on when it may be muddled by test or product code.
Its just a suggestion.